spec/BowlingSpec.js

```diff
@@ -0,0 +1,74 @@
+describe('Bowling', function () {
+
+  var myGlobal
+  var bowling
+  var f
+  var f2
```

Comment:
It feels like you don't need to declare all these frames here - they are only used in a single scope so why not make them local variables? (e.g. like on line 28)